torch/_inductor/codecache.py

```diff
@@ -98,6 +97,7 @@
 from torch.export.pt2_archive.constants import CUSTOM_OBJ_FILENAME_PREFIX
 from torch.fx.experimental.symbolic_shapes import has_hint, hint_int, ShapeEnv
 from torch.utils._ordered_set import OrderedSet
+from typing_extensions import override, Self
```

Comment:
Are you using the latest linter version? Why are there many linting differences?